src/OSPSuite.R/Services/ConcurrentSimulationRunner.cs

```diff
@@ -80,35 +111,52 @@ public void Clear()
             _cancellationTokenSource.Cancel();
```

Comment:
here you can use _cancellationTokenSource?.Cancel()

Reply:
True, done.